e2e_tests/test_shared_services.py

```diff
@@ -9,6 +9,7 @@
 LOGGER = logging.getLogger(__name__)
 
 
+@pytest.mark.extended
```

Comment:
Since almost every other service does an internal update on the firewall, do we even need it?

Reply:
Good point, I think we can remove this one